src/com/github/miachm/sods/LinkedValue.java

```diff
@@ -0,0 +1,87 @@
+package com.github.miachm.sods;
+
+import java.io.File;
+import java.net.URI;
+import java.net.URL;
+import java.util.Objects;
+
+/**
+ * Represents a value linked to a sheet, file, URI or URL.
+ */
+public class LinkedValue {
+    private final String href;
+    private final String value;
```

Comment:
Hi Miguel,

To address your comments:

 - You must have missed the fact that version was bumped with first commit, where bug fix was made; I am quite familiar with semantic versioning (https://semver.org/), I work with OSGi on a daily basis; you are free to bump it to whatever version you'd like; I needed to build the jar and include in local BND repo so I can continue on a different project where SODS library is consumed;
 - As it is mentioned in Javadoc of `LinkedValue` class, it is used to link a value to a sheet, file, URI or URL; this is a feature of ODS spec, perhaps you should check the generated XML code and/or generate an ODS file and open it with e.g. LibreOffice and you'll see how clicking a cell value with take you to another sheet, etc.;
 - I only need write capability, therefore I only focused on that; I do not really have the time to contribute more to this project at this point - that is all that is needed in a different project which consumes SODS library; if you or anyone else feels the need to add read capability, you are free to do so;
 - Separate class as well as builder pattern in `LinkedValue` was done on purpose, with hope that with time abstractions for other elements can also be created, a la https://github.com/jferard/fastods/, which is designed really well, unfortunately uses prohibitive license; currently, in SODS, everything is crammed together in `ODSWriter`, the elements do really know their XML representation, etc.;
 - `A1` is for compatibility with Excel, where sheet links written as `#SheetName` do not work in Excel, but `#SheetName.A1` work in both Excel and LibreOffice;
 - I am not really sure how to answer the asterisk question - I worked on this using IDEA IDE, and this is how it optimizes imports;
 - I did not have a timer on, but somewhere around 4+ hours, including time needed to analyze and come up with solution, plus additional integration test cases;


Feel free to use this as you'd like, I know I am already using this in a project where SODS is consumed. Sorry I do not have more time to help now.

Reply:
Don't get me wrong, I am grateful for any contribution. My comments try to balance between being appreciative and giving meaningful feedback according to the library standards. But sometimes I can excesively direct. I understand not everyone have the time to fully contribute to the development (dam, I don't even have the time myself).

Yep, some classes in SODS are becoming in god classes, specially ODSReader/ODSWritter like you mentioned. Nevertheless I try to keep the external-user API as consistent as posible, that's why the build pattern does not fit with the current implementation, because it presents to the user a completely different style. Eventually I'll need to deprecate the API and re-design it, but, you know, not enough time!

Thanks again, let's see if I can finish the feature and release another version.